tests/tui/conftest.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# conftest.py
+import os
+import shutil
+from pathlib import Path
+
+import pytest
+
+from halfpipe import resource
+from halfpipe.logging import logger
+from halfpipe.tui.base import MainApp  # Ensure path aligns with your project structure
+
+from .create_mock_bids_dataset import create_bids_data
+
+
+@pytest.fixture(scope="session", autouse=True)
+def resolved_test_dir_path():
+    """Fixture to resolve paths and handle fallback to './' if necessary."""
+    source_file = Path("/home/runner/actions-runner/_work/HALFpipe/HALFpipe/tests/tui/")
```

Comment:
Why is this necessary?

Reply:
on git it has to use hardcoded paths, and when running localy this path is not valid, so to be able smoothly run it localy but also on git I use that thing